docs/init_command.md

```diff
@@ -0,0 +1,65 @@
+# How to generate a configuration file
+
+To start quickly you can run the init command
+
+```bash
+vendor/bin/rector init
+```
+
+This will create a `rector.php` if it doesn´t already exist in your root directory with some sensitive defaults.
+
+```php
+// rector.php
+use Rector\Core\Configuration\Option;
+use Rector\Php74\Rector\Property\TypedPropertyRector;
+use Rector\Set\ValueObject\SetList;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // here we can define, what sets of rules will be applied
+    $parameters = $containerConfigurator->parameters();
+    $parameters->set(Option::SETS, [SetList::CODE_QUALITY]);
+
+    // register single rule
+    $services = $containerConfigurator->services();
+    $services->set(TypedPropertyRector::class);
+};
+```
+
+The init command takes an option called --template-type or in short form -t.
+If some other Rector extension like [rector-nette](https://github.com/rectorphp/rector-nette) or [rector-doctrine](https://github.com/rectorphp/rector-doctrine) provides such a custom template type you can specify it here:
+
+```bash
+vendor/bin/rector init --template-type=nette
+```
```

Comment:
Thanks for completing the answers!

One last detail - what is the motivation to use a "nette" template for `rector.php`? How does it differ from the main `rector.php`? So people understand, why they should consider using the `nette` one over generic one.

Reply:
Could we answer this later? Because i don´t know yet. I only know my personal motivation for TYPO3.

Reply:
I would have to come up with a contrived example because no template for nette or any other extension exists yet.

Reply:
I promise i will do that later.

Reply:
> I only know my personal motivation for TYPO3

Real motivation is the best. Put it there :+1:

Reply:
Done

Reply:
Awesome, now I get it! Thanks :+1: